devel/abseil/Portfile

```diff
@@ -18,23 +22,20 @@ long_description    Abseil is an open-source collection of C++ library \
                     Google's own C++ code base, has been extensively \
                     tested and used in production.
 
-if {${os.major} < 11} {
-
-    # NB this port builds with < darwin 11 if libcxx is installed
-    # with the emulated_tls variant to enable thread_local_storage
-
-    # https://abseil.io/docs/cpp/platforms/platforms
-    known_fail      yes
-    pre-fetch {
-        ui_error "${name} @${version} requires macOS 10.7 or later."
-        return -code error "incompatible OS X version"
-    }
-}
-
 checksums           rmd160  23e5de419df48b87661f55e2c2867a1d508793fc \
                     sha256  65d5a14a8e5745c15cdd4052b1d72f0d8e57d4b771441dead39479b1f786965c \
                     size    1957478
 
+if {${os.platform} eq "darwin" && ${os.major} < 11} {
+    if {${build_arch} in [list ppc ppc64]} {
+        patchfiles-append patch-darwin-ppc.diff
+    }
+    if {[string match *gcc* ${configure.compiler}]} {
+        configure.ldflags-append \
+                    -latomic
+    }
+}
```

Comment:
You might want to update this to use a `platform darwin {` block, rather than including that check within the list of conditions. (Not a big deal, but it simplifies the if statements.)

Reply:
Ok, fair enough, we can use GCC on >10 as well.